schema.json

```
@@ -97,6 +97,14 @@
                 "isBatchKeyASet": {
                     "type": "boolean",
                     "description": "(Optional) Set to true if the interface of the resource takes the batch key as a set (rather than an array). For example, when using a generated clientlib based on swagger where `uniqueItems: true` is set for the batchKey parameter. Default: false"
+                },
+                "propertyBatchKey": {
+                    "type": "string",
+                    "description": "The argument to the resource that represents the list of properties we want to fetch. (e.g. 'properties', 'features')"
```

the example here at first made me think that `'properties', 'features'` (as an array) was a valid example for `propertyBatchKey` in the config - like `propertyBatchKey: properties, features` lol

maybe to clarify:

```suggestion
                    "description": "(Optional) The argument to the resource that represents the nested list of optional properties we want to fetch. (e.g. usually 'properties' or 'features')"
```

---

++